recipes/bracken/meta.yaml

```diff
@@ -11,28 +11,30 @@ source:
   patches:
     - osx.patch  # [osx]
     - py3.patch
+    - ctime.patch
 
 build:
-  number: 2
+  number: 0
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - llvm-openmp  # [osx]
+    - libgomp      # [linux]
   host:
     - python
   run:
     - python
     - kraken2
     - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```

Comment:
I think that is not needed, as we have a run_exports defined: https://github.com/conda-forge/ctng-compilers-feedstock/blob/9c214ed1ca8e16813e056243ef0c8bd65e054630/recipe/meta.yaml#L330

Reply:
I was copying this from MMseqs2 where @dpryan79 explicitly added both:
https://github.com/bioconda/bioconda-recipes/commit/a4001e18c0fdd964c9e9a60a1c47fb8da516a7b2#diff-2d85eb374c8cebc33fadbc998ec527e768d823ee8473bca28cfe16ac268f8c35

It was compiling correctly on Linux without the explicit dependency though. Should I remove it again?

Reply:
But Devan has also only added it to the build section, not the run section, left?

Reply:
So is `llvm-openmp` in run also needed?

Reply:
I removed both lets see what happens

Reply:
It seems to also have a run_export: https://github.com/conda-forge/openmp-feedstock/blob/master/recipe/meta.yaml#L31